### ISSUE TITLE
WOfS Jagged edge effect

### ISSUE THREAD

**Issue by pfrost2611** [GeoscienceAustralia/wofs#37]
Hi, we have upgraded the latest WOfS version and applied it on both S2 and Landsat C2 data. Has anyone else observed the jagged edge effect produced along all water/land boundaries?

![Jagged_effect](https://user-images.githubusercontent.com/58635846/111144074-3f139e80-858f-11eb-96d2-6576dc61d4d0.png)

**Comment by robbibt** [comment]
This looks like what you recently encountered @MatthewJA? It is likely a resampling effect, either due to the way the underlying DEM is resampled or resampling of the SWIR bands. I'm not familiar with the WOFS codebase personally, but is there an option to configure resampling methods in the code? If so, `bilinear` would be a better choice than `nearest`.

**Comment by MatthewJA** [comment]
Yep, this is a resampling effect (to some extent); resample your data with bilinear resampling before you run WOfS on it.
